app/js/controllers/main.js

```diff
@@ -9,16 +8,23 @@ import DevController from './dev';
 
 import Foxbox from '../lib/foxbox/foxbox';
 
-export default class MainController extends RoutingController {
+const p = Object.freeze({
```

Comment:
Have you considered moving the router to [react-router](https://github.com/reactjs/react-router)?
I think we should use it in the future, but this approach works for now.